### PR TITLE
Updated build instructions and ReadMe

### PR DESCRIPTION
**Pull request by liealex**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,23 +10,7 @@ Platform details : [![Hardware](https://img.shields.io/badge/hardware-wiki-green
 
 ## Supported Host Platforms
 
-For more details on building the SDK on a host platform please check the **User Guide** specified below:.
-
-| Operating System | Documentation | GitHub main status | Downloads |
-| --------- | ----------- | ----------- | ----------- |
-| Windows | [Build Instructions](scripts/windows) | [![Build status](https://dev.azure.com/AnalogDevices/3DToF-rework/_apis/build/status/analogdevicesinc.ToF?branchName=main)](https://dev.azure.com/AnalogDevices/3DToF-rework/_build?view=runs&branchFilter=3310) | [![ToF evaluation kit installer](https://img.shields.io/badge/release-ToF_evaluation_kit_installer-blue.svg)](https://github.com/analogdevicesinc/ToF/releases/latest) |
-| Linux | [Build Instructions](doc/itof/linux_build_instructions.md) | [![Build status](https://dev.azure.com/AnalogDevices/3DToF-rework/_apis/build/status/analogdevicesinc.ToF?branchName=main)](https://dev.azure.com/AnalogDevices/3DToF-rework/_build?view=runs&branchFilter=3310) | [![ToF evaluation kit installer](https://img.shields.io/badge/release-ToF_evaluation_kit_installer-blue.svg)](https://github.com/analogdevicesinc/ToF/releases/latest) |
```
Comment (dNechita) [inline README.md:18]: This table makes sense to exists as the SDK can be built on different platforms. Showing which platforms (or operating systems) are supported is valuable and so are the status of the CI jobs for that platforms.

**Comment by dNechita** [comment]
As a general guideline, while it makes sense for ToF to reference things from libaditof (links to docs, readmes, build steps, etc.) it doesn't make much sense for libaditof to reference things from ToF. If it really must, it should be kept to a minimum. The reason is that libaditof is a part of ToF while Tof is not a part of libaditof.

File: README.md (hunk 1 above)
Comment (liealex) [reply README.md]: Solved

**Comment by liealex** [comment]
> As a general guideline, while it makes sense for ToF to reference things from libaditof (links to docs, readmes, build steps, etc.) it doesn't make much sense for libaditof to reference things from ToF. If it really must, it should be kept to a minimum. The reason is that libaditof is a part of ToF while Tof is not a part of libaditof.

In this case, should this part be removed? 
https://github.com/analogdevicesinc/libaditof/pull/4/files#diff-00ff7d396af4541ffe510305e505ba8c9148d43db7de1ec9d5622f57df8fe125R112

**Comment by dNechita** [comment]
> > As a general guideline, while it makes sense for ToF to reference things from libaditof (links to docs, readmes, build steps, etc.) it doesn't make much sense for libaditof to reference things from ToF. If it really must, it should be kept to a minimum. The reason is that libaditof is a part of ToF while Tof is not a part of libaditof.
> 
> In this case, should this part be removed? https://github.com/analogdevicesinc/libaditof/pull/4/files#diff-00ff7d396af4541ffe510305e505ba8c9148d43db7de1ec9d5622f57df8fe125R112

I think this is one exception. So it can be kept.

**Review by dNechita — APPROVED** [review]

**Pull request merged**